diary.cfg

```
@@ -0,0 +1,10 @@
+# Path that holds the journal text files
+#dir = ~/diary
+# Number of years to show before/after todays date
+range = 1
+# 0 = Sunday, 1 = Monday, ..., 6 = Saturday
+weekday = 1
+# Date and file format, change with care
+fmt = %Y-%m-%d
+# Editor to open journal files with
+editor = ""
```

```suggestion
editor = "nano"
```

May as well make it useful and friendly to beginners

---

That's a good idea. But unfortunately it would destroy the nice read-only side-effect of not setting any editor (the pristine default behavior, so to say). Let's leave this as is for now.